# Disable use of orderfiles which are difficult to generate and normally require downloading from Google storage

--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -424,22 +424,6 @@ if (is_win) {
         "//content/public/app:both",
       ]
     }
-
-    if (is_clang && is_official_build) {
-      orderfile = "build/chrome.$target_cpu.orderfile"
-      rebased_orderfile = rebase_path(orderfile, root_build_dir)
-      inputs = [
-        orderfile,
-      ]
-      ldflags += [
-        "/order:@$rebased_orderfile",
-
-        # Ignore warnings about missing functions or functions not in their
-        # own section.
-        "/ignore:4037",
-        "/ignore:4065",
-      ]
-    }
   }
 
   if (is_multi_dll_chrome) {
@@ -486,22 +470,6 @@ if (is_win) {
         "/DELAYLOAD:wininet.dll",
       ]
 
-      if (is_clang && is_official_build) {
-        orderfile = "build/chrome_child.$target_cpu.orderfile"
-        rebased_orderfile = rebase_path(orderfile, root_build_dir)
-        inputs = [
-          orderfile,
-        ]
-        ldflags += [
-          "/order:@$rebased_orderfile",
-
-          # Ignore warnings about missing functions or functions not in their
-          # own section.
-          "/ignore:4037",
-          "/ignore:4065",
-        ]
-      }
-
       if (symbol_level == 2) {
         # Incremental linking doesn't work on this target in debug mode with
         # full symbols, but does work in other cases, including minimal
